test/client/karma.conf.js

```diff
@@ -14,35 +14,35 @@ const launchers = {
     browser: 'firefox',
     os: 'Windows',
     os_version: '10'
-  },
-  bs_safari: {
-    base: 'BrowserStack',
-    browser: 'safari',
-    browser_version: '9.0',
-    os_version: 'El Capitan',
-    os: 'OS X'
-  },
-  bs_ie_11: {
-    base: 'BrowserStack',
-    browser: 'ie',
-    browser_version: '11.0',
-    os: 'Windows',
-    os_version: '10'
-  },
-  bs_ie_10: {
-    base: 'BrowserStack',
-    browser: 'ie',
-    browser_version: '10.0',
-    os: 'Windows',
-    os_version: '8'
-  },
-  bs_ie_9: {
-    base: 'BrowserStack',
-    browser: 'ie',
-    browser_version: '9.0',
-    os: 'Windows',
-    os_version: '7'
   }
+  // bs_safari: {
+  //   base: 'BrowserStack',
+  //   browser: 'safari',
+  //   browser_version: '9.0',
+  //   os_version: 'El Capitan',
+  //   os: 'OS X'
+  // },
+  // bs_ie_11: {
+  //   base: 'BrowserStack',
+  //   browser: 'ie',
+  //   browser_version: '11.0',
+  //   os: 'Windows',
+  //   os_version: '10'
+  // },
+  // bs_ie_10: {
+  //   base: 'BrowserStack',
+  //   browser: 'ie',
+  //   browser_version: '10.0',
+  //   os: 'Windows',
+  //   os_version: '8'
+  // },
+  // bs_ie_9: {
+  //   base: 'BrowserStack',
+  //   browser: 'ie',
+  //   browser_version: '9.0',
+  //   os: 'Windows',
+  //   os_version: '7'
+  // }
```

Comment:
Why did you left them commented out? I think it is better to remove them altogether. Right?

Reply:
I'm leaving these in because I don't understand the browserstack configuration and I have not been able to find any alternative examples.  We should try again to fix this but if we fail we can remove the commented code.